report-hazard/index.html

```diff
@@ -0,0 +1,141 @@
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav>
+      <ul>
+        <li><a title="Select the location" href="#step1">1</a></li>
+
+        <li>
+          <a title="What type of hazard would you like to report?" href="#step2"
+            >2</a
+          >
+        </li>
+
+        <li>
+          <a title="What type of wild animal did you spot?" href="#step3">3</a>
+        </li>
+
+        <li>
+          <a
+            title="Would you like to mention any further details?"
+            href="#step4"
+            >4</a
+          >
+        </li>
+
+        <li>
+          <a
+            title="Please upload photos to help others see the reported hazard visually"
+            href="#step5"
+            >5</a
+          >
+        </li>
+
+        <li>
+          <a title="Review the report before submitting." href="#step6">6</a>
+        </li>
+
+        <li><a href="#step7">7</a></li>
+      </ul>
+    </nav>
+
+    <main>
+      <form id="form">
+        <section class="page" id="step1">
+          <h1>1. Select the location.</h1>
+
+          <div id="map"></div>
+
+          <label>Your current location is:</label>
+          <input type="text" id="name" required />
+
+          <a href="#step2">Continue</a>
+        </section>
+        <section class="page" id="step2">
+          <h1>2. What type of hazard would you like to report?</h1>
+          <input type="radio" name="category" value="Wildlife" />Wildlife
+          <input type="radio" name="category" value="Weather" />Weather
+          <input
+            type="radio"
+            name="category"
+            value="Infrastructure"
+          />Infrastructure
+          <input type="radio" name="category" value="Campsite" />Campsite
+          <a href="#step1">Back</a>
+          <a href="#step3">Continue</a>
+        </section>
+        <section class="page" id="step3">
+          <h1>3. What type of wild animal did you spot?</h1>
+          <input type="radio" name="type" value="Bear" />Bear
+          <input type="radio" name="type" value="Coyote" />Coyote
+          <input type="radio" name="type" value="Coguar" />Coguar
+          <input type="radio" name="type" value="Elk" />Elk
+          <input type="radio" name="type" value="Raccoon" />Raccoon
+          <a href="#step2">Back</a>
+          <a href="#step4">Continue</a>
+        </section>
+
+        <section class="page" id="step4">
+          <h1>4. Would you like to mention any further details?</h1>
+          <input type="text" id="details" />
+
+          <a href="#step3">Back</a>
+          <a href="#step5">Continue</a>
+        </section>
+
+        <section class="page" id="step5">
+          <h1>
+            5. Please upload photos to help others see the reported hazard
+            visually.
+          </h1>
+          <p>Upload pictures</p>
+          <input type="text" id="picture" />
+
+          <a href="#step4">Back</a>
+          <a href="#step6" onClick="myFu()">Continue</a>
```

Comment:
let's keep the onClick references in the javascript using the id of the element and the addEventListener

instead of 
 <a href="#step6" onClick="myFu()">Continue</a>
 do
  <a href="#step6" id="submitReportBtn">Continue</a>